devops/db/liquibase/changelogs/payment-transaction-gateway/incremental-changelogs/V7_PM_1215-changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         https://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.8.xsd">
+    <changeSet id="V13__PM_1215_ADD_IS_FIRST_PAYMENT_AND_COLUMN_RESPONSE_TYPE" author="liquibase" failOnError="true" context="incremental">
+        <sqlFile path="../scripts/table-creation/V13__PM_1215_ADD_IS_FIRST_PAYMENT_AND_COLUMN_RESPONSE_TYPE.sql" stripComments="true" relativeToChangelogFile="true" dbms="oracle" encoding="UTF-8" />
+    </changeSet>
+</databaseChangeLog>
```

Comment:
Sanity check: do these changes apply to the postgres migration too?

Reply:
I merged this commit from another branch just to be up to date.